package.json

```diff
@@ -35,6 +35,9 @@
     "ufo": "^1.0.1",
     "uncrypto": "^0.1.2"
   },
+  "peerDependencies": {
+    "@nozomuikuta/h3-cors": "^0.2.1"
+  },
```

Comment:
Should I move `@nozomuikuta/h3-cors` to `dependencies` (cf. CI error on build step)?